google/cloud/spanner/numeric.cc

```diff
@@ -31,15 +31,17 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 
 namespace {
 
-inline bool IsDigit(char ch) {
+inline bool IsDigit(unsigned char ch) {
   return std::isdigit(static_cast<unsigned char>(ch)) != 0;
 }
 
-inline bool IsSpace(char ch) {
+inline bool IsSpace(unsigned char ch) {
   return std::isspace(static_cast<unsigned char>(ch)) != 0;
 }
```

Comment:
These ones look fine because they already have the cast in the body.

Reply:
Ooops. Fixed.